### ISSUE TITLE
Meeting ideas and todos

### ISSUE THREAD

**Issue by sudo-Boris** [timherzig/decomposition_learning#12]
### Ideas

- [ ] clip gradients
- [ ] remove skip connections between SWIN encoder and UNet decoder
- [ ] try diffusion with pre-trained SWIN
- [ ] train with weight decay

### General Todos
- [ ] get gradient logging to work
- [ ] merge most branches into main
